source/Nuke.Common/CI/ShutdownDotNetAfterServerBuildAttribute.cs

```diff
@@ -22,7 +22,7 @@ public class ShutdownDotNetAfterServerBuildAttribute : BuildExtensionAttributeBa
 
         public void OnBuildFinished(NukeBuild build)
         {
-            if (NukeBuild.IsServerBuild)
+            if (NukeBuild.IsServerBuild && NukeBuild.Host is not NukeInDocker)
```

Comment:
I was seeing errors 
```
Unhandled exception. Nuke.Common.Tooling.ProcessException: Process 'dotnet' exited with code 145.
   > /usr/bin/dotnet build-server shutdown
   @ /build
Error output:
   Could not execute because the application was not found or a compatible .NET SDK is not installed.
   Possible reasons for this include:
     * You intended to execute a .NET program:
         The application 'build-server' does not exist.
     * You intended to execute a .NET SDK command:
         A compatible installed .NET SDK for global.json version [6.0.100] from [/build/global.json] was not found.
         Install the [6.0.100] .NET SDK or update [/build/global.json] with an installed .NET SDK:

   at Nuke.Common.Tooling.ProcessExtensions.AssertZeroExitCode(IProcess process) in C:\Data\dev\Octopus\spikes\nuke\source\Nuke.Common\Tooling\ProcessExtensions.cs:line 39
   at Nuke.Common.Tools.DotNet.DotNetTasks.DotNet(String arguments, String workingDirectory, IReadOnlyDictionary`2 environmentVariables, Nullable`1 timeout, Nullable`1 logOutput, Nullable`1 logInvocation, Func`2 outputFilter) in C:\Data\dev\Octopus\spikes\nuke\source\Nuke.Common\Tools\DotNet\DotNet.Generated.cs:line 41
   at Nuke.Common.CI.ShutdownDotNetAfterServerBuildAttribute.OnBuildFinished(NukeBuild build) in C:\Data\dev\Octopus\spikes\nuke\source\Nuke.Common\CI\ShutdownDotNetAfterServerBuildAttribute.cs:line 26
   at Nuke.Common.Utilities.Collections.EnumerableExtensions.ForEach[T](IEnumerable`1 enumerable, Action`1 action) in C:\Data\dev\Octopus\spikes\nuke\source\Nuke.Common\Utilities\Collections\Enumerable.ForEach.cs:line 17
   at Nuke.Common.NukeBuild.ExecuteExtension[TExtension](Expression`1 action)
   at Nuke.Common.Execution.BuildManager.Execute[T](Expression`1[] defaultTargetExpressions) in C:\Data\dev\Octopus\spikes\nuke\source\Nuke.Common\Execution\BuildManager.cs:line 81
   at Nuke.Common.NukeBuild.Execute[T](Expression`1[] defaultTargetExpressions) in C:\Data\dev\Octopus\spikes\nuke\source\Nuke.Common\NukeBuild.cs:line 73
   at Build.Main() in C:\Data\dev\Octopus\spikes\nuke\build\Build.cs:line 52
```
from within the container - as we're killing the container, we dont need it.